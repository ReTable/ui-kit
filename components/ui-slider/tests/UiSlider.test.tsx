import { create } from 'react-test-renderer';
import { describe, expect, it } from 'vitest';

import { UiSlider } from '~';

describe('UiSlider', () => {
  it('works', () => {
    const tree = create(<UiSlider />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
