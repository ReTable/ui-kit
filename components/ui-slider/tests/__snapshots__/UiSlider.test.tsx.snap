// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`UiSlider > works 1`] = `
<div
  className="tabula_ui_slider__b6eibh0"
/>
`;
